src/pages/projects/form/ContributorRow.tsx

```diff
@@ -0,0 +1,230 @@
+                  </li>
+                );
+              }}
+              renderInput={(params) => (
+                <AutocompleteTextField
+                  {...params}
+                  onBlur={() => setFieldTouched(`${field.name}.id`)}
+                  value={field.value}
+                  name={field.name}
+                  data-testid={dataTestId.registrationWizard.description.projectForm.contributorsSearchField}
+                  required
+                  label={t('project.person')}
+                  placeholder={t('project.form.search_for_person')}
+                  errorMessage={
+                    touched.contributors?.[contributorIndex]?.identity?.id && !!contributorErrors?.identity?.id
+                      ? contributorErrors?.identity?.id
+                      : ''
+                  }
+                  isLoading={personQuery.isFetching}
+                  showSearchIcon={!field.value.id}
+                />
+              )}
+            />
+          )}
+        </Field>
+      </TableCell>
+      <TableCell>
+        <>
+          {contributor.roles
+            .filter((r) => r.affiliation.id)
+            .map((role) => (
+              <ProjectOrganizationBox
+                key={role.affiliation.id}
+                unitUri={role.affiliation.id}
+                authorName={getFullName(contributor.identity.firstName, contributor.identity.lastName)}
+                contributorRoles={contributor.roles}
+                baseFieldName={baseFieldRoles}
+                sx={{ width: '100%', marginBottom: '0.5rem' }}
+                removeAffiliation={
+                  contributor.roles.length === 1 || role.type === 'ProjectManager'
+                    ? undefined
+                    : () => removeAffiliation(role.affiliation.id)
+                }
+              />
+            ))}
+          <Button
+            variant="outlined"
+            sx={{ padding: '0.1rem 0.75rem', marginTop: '0.5rem', marginBottom: '0.5rem' }}
+            data-testid={dataTestId.registrationWizard.contributors.addAffiliationButton}
+            startIcon={<AddIcon />}
+            color={affiliationFieldTouched && affiliationError ? 'error' : 'inherit'}
+            onClick={toggleAffiliationModal}>
+            {t('project.add_affiliation')}
+          </Button>
+          {affiliationFieldTouched && affiliationError && (
+            <Typography sx={{ color: 'error.main', marginTop: '0.25rem', letterSpacing: '0.03333em' }}>
+              {affiliationError}
+            </Typography>
+          )}
+        </>
+      </TableCell>
+      <TableCell sx={{ width: '3rem', textAlign: 'center', paddingTop: '1rem' }}>
+        <DeleteIconButton
+          data-testid={dataTestId.registrationWizard.description.projectForm.removeContributorButton}
+          onClick={() => setShowConfirmRemoveContributor(true)}
+          tooltip={t('project.form.remove_participant')}
+          disabled={!removeContributor}
```

Comment:
Ser ut som denne alltid er disabled for det første elementet, mens den skulle vært disabled om det bare er ett element?
![bilde](https://github.com/user-attachments/assets/c2cacd6c-d8ee-482f-a6b8-92685d4da754)

Reply:
Nei, logikken er kodet slik at den også er disablet hvis det er rollen til projectmanager.

Reply:
Mulig NTNU kunne rykket opp til å bli project manager-rolle, men det er litt skummelt å gjøre så mye logikk som ikke vises i frontend syns jeg

Reply:
Ah ja, for nå ligger jeg inne som prosjektleder i Sikt, mens jeg er prosjektdeltager i NTNU?
Det tror jeg egentlig er feil da, og at i dette tilfelle er jeg bare prosjektleder (ikke både leder og deltager), men jeg er det på vegne av to ulike enheter.
Det beste hadde kanskje vært å bruke samme rolle som for den første tilknytningen også for alle andre tilknytninger
Men om vi har en plan for å løse dette i wizarden er det greit for meg

Reply:
Jeg spurte ihvertfall JE om det var greit at rolle nr to alltid var "Prosjektdeltaker" og han sa ja på det og at majoriteten alltid skulle være prosjektdeltaker.